library/tutorials/inheritFrom/0.json

```diff
@@ -820,9 +829,10 @@
             "y": 0,
             "width": 103,
             "height": 160,
-            "valueType": "static",
-            "value": "/i/cards-default/2B.svg",
-            "color": "transparent"
+            "color": "transparent",
+            "dynamicProperties": {
+              "value": "/i/cards-default/2B.svg"
```

Comment:
How did this happen? It was static but now it is dynamic and it doesn't look like a property name. Was this the file updater or a manual change?

Reply:
No idea.  It isn't that way in the file that Raphael linked.  I just downloaded that and uploaded it to Git.  Don't think I touched anything.  Has to have been a file updater.

Reply:
File updater. I don't recall changing anything on cards